core/src/main/java/org/kiwix/kiwixmobile/core/settings/SliderPreference.java

```diff
@@ -168,12 +174,22 @@ protected void onBindView(View view) {
   private void setGrayState(View view) {
     SharedPreferenceUtil sharedPreferenceUtil = new SharedPreferenceUtil(getContext());
     boolean enabled = sharedPreferenceUtil.getPrefZoomEnabled();
+    NightModeConfig nightMode = new NightModeConfig(sharedPreferenceUtil,getContext());
+    boolean nightMode_enabled = nightMode.isNightModeActive();
 
     TextView titleView = view.findViewById(android.R.id.title);
     TextView summaryTV = view.findViewById(android.R.id.summary);
     if (!enabled) {
```

Comment:
As you have appended `else` the negative control flow can be avoided:
```java
if (enabled) {
  // some statements
} else {
  // other statements
}
```
Negative flow is difficult to read. :(